sdk/storage/CONTRIBUTING.md

```diff
@@ -50,10 +50,7 @@ exercise any new features that you've added.
 
 We use [nUnit 3](https://github.com/nunit/docs/wiki) as our testing framework.
```

Comment:
nit:  Not your change, but the product name is `NUnit`.   Should we fix this while we're making changes?